src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAO.scala

```diff
@@ -89,28 +89,30 @@ class GraphAuthDAO extends AuthDAO with GraphDAO {
       // user -> group -> ...some other groups... -> group -> workspace access map
       //Multiple paths implies that the user belongs to multiple groups associated with the workspace.
       val accessMapPaths =
-        userPipeline(db, user).as("vtx").in()
+        userPipeline(db, user).as("vtx").inE().outV()
```

Comment:
did this to capture the edges between workspace and acl maps, acl maps and group more effectively